symengine/add.cpp

```diff
@@ -290,6 +290,35 @@ RCP<const Basic> add(const RCP<const Basic> &a, const RCP<const Basic> &b)
     return Add::from_dict(coef, std::move(d));
 }
 
+RCP<const Basic> add(const vec_basic &a)
+{
+    if (a.size() < 2) {
+        throw std::runtime_error("add expected at least a vector with size two");
```

Comment:
We can do `add({i2})` but there should be at-least two `RCP<const Basic>` to add, it seems little obscure giving one `RCP<const Basic>` to add.